admin/includes/functions/general.php

```diff
@@ -2575,21 +2566,6 @@ function zen_geo_zones_pull_down_coupon($parameters, $selected = '') {
     return $select_string;
   }
 
-/**
- * customer lookup of address book
- */
-  function zen_get_customers_address_book($customer_id) {
-    global $db;
-
-    $customer_address_book_count_query = "SELECT c.*, ab.* from " .
-                                          TABLE_CUSTOMERS . " c
-                                          left join " . TABLE_ADDRESS_BOOK . " ab on c.customers_id = ab.customers_id
-                                          WHERE c.customers_id = " . (int)$customer_id;
-
-    $customer_address_book_count = $db->Execute($customer_address_book_count_query);
-    return $customer_address_book_count;
-  }
-
 /**
```

Comment:
Perhaps preserve the interfaces for these old functions but use the new class under the hood?

Reply:
Yes. It's in the functions_customers file.